README.md

```diff
@@ -402,6 +406,91 @@ Push this configration to CipherStash:
 stash upload-config --file dataset.yml --client-id $CS_CLIENT_ID --client-key $CS_CLIENT_KEY
 ```
 
+_NOTE:_
+
+_The Rails demo app uses Active Admin, which uses a dependency called Ransack for the filters on the side of the dashboard view._
+
+_So these filters continue to work when reading encrypted fields, the below needs to be added the Rails app._
+
+Add this to the `patients.rb` file:
+
+```ruby
+# app/admin/patients.rb
```

Comment:
I'm starting to wonder if it'd be easier to check in some of these changes and include them from the starting point.

We could have some comments in the source files that explain some of the active admin details for those that are curious. That way we can highlight the important parts that are most relevant to the demo in the README and leave out some of the active admin tedium.

Reply:
@CDThomas that's a really good point. Way easier to uncomment than copy-paste.

Reply:
@CDThomas @auxesis I thought of doing that, but if they try and run the string filters in `plaintext-duplicate` mode, they will not work. Because the custom functions we add, to keep the string filters working in encrypted-duplicate and encrypted mode, strip out the`%` from the like queries.

Reply:
Oh, you mean just add it, but commented out?

Reply:
I'll add it into the demo, commented out with notes.

I'll need to add something in the README around it though, to point them to uncomment it.

Reply:
> Oh, you mean just add it, but commented out?

Ah, I meant have any active-admin related code included in the project from the start (not commented out), but I didn't think about the `LIKE` query issues.

Commenting it out might work though.

I'm really tempted to get `LIKE` queries working for both `plaintext-duplicate` and `enrypted*` modes without editing files. If we don't want to deal with making the actual `LIKE` implementation more robust yet, I wonder if it'd be straightforward to handle it in the match indexer instead. We could add support for `char_filters` and strip out `%`s and `_`s or something.

Reply:
> We could add support for char_filters and strip out `%`s and `_`s or something.

Since you added support for `ILIKE` that might mean that we wouldn't even need the custom active admin filter methods, too @fimac.

Reply:
We still need them, because the filters send through the ILIKE query with the `%`, so the match query won't work. So we need to intercept the query anyway, so we're sending through just the value. Unless i'm missing something with how the match query is implemented?

Reply:
> I'm really tempted to get `LIKE` queries working for both `plaintext-duplicate` and `enrypted*` modes without editing files. If we don't want to deal with making the actual `LIKE` implementation more robust yet, I wonder if it'd be straightforward to handle it in the match indexer instead. We could add support for `char_filters` and strip out `%`s and `_`s or something.

This would be ideal, I can create a card if there isn't one already?

Reply:
`char_filter` support seems like something we could want in the long term anyway, so seems reasonable to me. It feels less hacky than stripping out lead/trailing `%`s in the driver code.

Maybe run it by James and/or Dan too and card it up if it sounds good to them. I don't think it'd be too hard to implement.